addons/sys_core/fnc_getVOIPChannelUID.sqf

```diff
@@ -0,0 +1,18 @@
+#include "script_component.hpp"
+/*
+ * Author: Killerswin2
+ * Returns the VOIP channel UID.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * 0: VOIP Channel UID <STRING>
+ *
+ * Example:
+ * [] call acre_sys_core_fnc_getVOIPChannelUID
+ *
+ * Public: No
+ */
+
+GVAR(ichannelUID)
```

Comment:
```suggestion
GVAR(iChannelUID)
```
Presumably this capitalization was the intended one, as used in `acre_sys_core_fnc_handleGetVOIPChannelUID` below.

Reply:
Yeah that is the case.